cs-utilities/pom.xml

```diff
@@ -60,16 +68,18 @@
         <maven-release-plugin.version>2.5.2</maven-release-plugin.version>
         <maven-javadoc-plugin.version>2.10.4</maven-javadoc-plugin.version>
         <maven-source-plugin.version>3.0.1</maven-source-plugin.version>
-        <maven-delpoy-plugin.version>2.8.2</maven-delpoy-plugin.version>
+        <maven-deploy-plugin.version>2.8.2</maven-deploy-plugin.version>
         <maven-gpg-plugin.version>1.5</maven-gpg-plugin.version>
         <!--Dependencies versions-->
         <score-content-sdk.version>1.10.6</score-content-sdk.version>
         <cs-commons.version>0.0.4</cs-commons.version>
         <junit.version>4.12</junit.version>
+        <pdfbox.version>2.0.8</pdfbox.version>
         <!--Misc properties-->
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.compiler.source>1.7</maven.compiler.source>
         <maven.compiler.target>1.7</maven.compiler.target>
+        <custom.nexus>overwriten_by_ci</custom.nexus>
```

Comment:
is this overwritten in some cases ?

Reply:
It will be overwritten by the Jenkins job.